.env.tpl

```diff
@@ -37,8 +37,6 @@ KLANTINTERACTIES_API_CLIENT_MP_REST_URL=http://localhost:8002
 KLANTINTERACTIES_API_TOKEN=dummyToken
 KVK_API_CLIENT_MP_REST_URL=op://Dimpact/ZAC-.env-$APP_ENV/KVK/API_CLIENT_MP_REST_URL
 KVK_API_KEY=op://Dimpact/ZAC-.env-$APP_ENV/KVK/API_KEY
-MAILJET_API_KEY=op://Dimpact/ZAC-.env-$APP_ENV/MAILJET/API_KEY
```

Comment:
Why remove the Mailjet keys here? We need Mailjet for local development no? Without these settings does sending mail still work locally?

Reply:
Yes it works. We have the same expression for `SMTP_USERNAME` and `SMTP_PASSWORD`. The Mailjet ones were not used.

Reply:
ok! but I also see them in `.env.example` and in our `docker-compose.yaml`. Shouldn't they be removed there as well?